examples/src/main/java/io/kubernetes/client/examples/ExecExample.java

```diff
@@ -98,6 +98,19 @@ public void run() {
             });
     out.start();
 
+    Thread error =
```

Comment:
this looks irrelevant w/ the purpose of this pull? but the change is good

Reply:
I will move this to a different PR.